scrapers/scraper-py-dillons/dillons-scraper.py

```diff
@@ -0,0 +1,177 @@
+import requests
+import json
+import time
+
+# --------------------------------------------------
+# Kroger Developer Credentials
+# --------------------------------------------------
+CLIENT_ID = "grocerywise-24326124303424486d71346d664548627957596f68623757642e4230656f614a6a424357474d7752417551314d797242694b4a74514f696d5a4c366d1253636185741314204"
+CLIENT_SECRET = "LrShuNpEplhS9lJSUbON3qbXJEIlfrHKy21cuWku"
```

Comment:
Move this into a .env,

.env will not be tracked by github

Reply:
Be sure to regenerate the client secret, Dillons will probably have revoked it since it is online now. Make a .env-sample, message the client secret in other avenue